ext/serializer.c

```diff
@@ -1210,17 +1210,28 @@ void ddtrace_serialize_span_to_array(ddtrace_span_data *span, zval *array) {
     add_assoc_long(el, "start", span->start);
     add_assoc_long(el, "duration", span->duration);
 
+    zend_array *meta = ddtrace_property_array(&span->property_meta);
+
     // SpanData::$name defaults to fully qualified called name (set at span close)
-    zval *prop_name = &span->property_name;
+    zval *operation_name = zend_hash_str_find(meta, ZEND_STRL("operation.name"));
+    if (operation_name) {
+        zend_str_tolower(Z_STRVAL_P(operation_name), Z_STRLEN_P(operation_name));
```

Comment:
Use `zend_string_tolower`, this makes a copy if necessary (which you'll have to release then, too). Lowering operation_name directly is problematic if the value isn't RC=1.